include/aspect/simulator.h

```diff
@@ -431,6 +431,91 @@ namespace aspect
       void solve_timestep ();
 
       /**
+       * This function implements one scheme for the various
+       * steps necessary to assemble and solve the nonlinear problem.
+       *
+       * 'IMPES' is the classical IMplicit
+       * Pressure Explicit Saturation scheme in which ones solves
+       * the temperatures and Stokes equations exactly
+       * once per time step, one after the other.
```

Comment:
What a shame that we didn't give this a better name -- because what we actually implement these days is an IMPIS scheme since we also treat the temperature equations implicitly :-(